browser/src/libs/phabricator/dom_functions.ts

```diff
@@ -79,21 +79,27 @@ export const diffDomFunctions: DOMFunctions = {
         }
 
         const td = target.closest('td')
-        if (
-            td &&
-            (td.classList.contains('show-more') ||
-                td.classList.contains('show-context') ||
-                !getLineNumberCellFromCodeElement(td))
-        ) {
+        if (!td) {
+            return null
+        }
+        if (td.classList.contains('show-more') || td.classList.contains('show-context')) {
+            // This element represents a collapsed part of the diff, it's not a code element.
+            return null
+        }
+        if (getLineNumberCellFromCodeElement(td)) {
```

Comment:
```suggestion
        if (!getLineNumberCellFromCodeElement(td)) {
```

Did you mean to negate this?